src/main/java/Handler/FileHandler.java

```diff
@@ -11,29 +13,39 @@ public Response generateResponse(Request request) {
      
     switch (request.getMethod()) { 
       case "GET":  
-        return buildGETResponse(); 
+        return buildGetResponse(); 
       default:  
         return new Response.Builder(HttpStatusCode.METHOD_NOT_ALLOWED) 
                            .build(); 
     } 
   }
 
-  private Response buildGETResponse() { 
+  private Response buildGetResponse() {
     int statusCode = determineStatusCode();
-    String messageBody = createMessageBody(statusCode);
-     
-    return new Response.Builder(statusCode) 
-                       .messageBody(messageBody) 
-                       .build(); 
-  } 
+    String messageBody = createMessageBody();
+    int contentLength = ResponseHeader.determineContentLength(messageBody);
```

Comment:
I'm sure I'll find this shortly after I add this comment, but I don't understand what `ResponseHeader` is - I assume it's a class with static methods but I don't see it imported

Reply:
I'm not sure if this is something that Gradle handles, but I hadn't had to import any classes for my code to work. Should I be importing a class any time it's referenced inside of another class?